README.rst

```diff
@@ -92,3 +92,9 @@ You want to tell us about it – best of all!
 Start at the `contributing guide <http://matplotlib.org/devdocs/devel/contributing.html>`_!
 
 Developer notes are now at `Developer Discussions <https://github.com/orgs/matplotlib/teams/developers/discussions>`_ (Note: For technical reasons, this is currently only accessible for matplotlib developers.)
+
+Citing Matplotlib
+==========
```

Comment:
I think this underline (equal signs) would be better as:
```rst
Citing Matplotlib
=================
```

Reply:
@bingyao done